indigo_app/notifications.py

```diff
@@ -115,6 +140,12 @@ def notify_comment_posted(comment_id):
         log.warning("Comment with id {} doesn't exist, ignoring".format(comment_id))
 
 
+@receiver(post_save, sender=Annotation)
+def post_annotation_reply(sender, **kwargs):
+    if kwargs['instance'].in_reply_to:
+        notifier.notify_reply_to_annotation(kwargs['instance'].in_reply_to, kwargs['instance'])
```

Comment:
Minor comment: I think it makes more sense for `notify_reply_to_annotation` to simply receive the entire annotation object. That has all the context, including the parent. It can then decide what to do. In other words, passing in the parent AND the annotation is unnecessary.

Reply:
Yes, it does. Rectifying this